frontend/src/Editor/QueryManager/QueryEditors/TooljetDatabase/util.js

```diff
@@ -28,7 +28,7 @@ export const hasEqualWithNull = (queryOptions, operation) => {
     const filterKeys = Object.keys(filters);
     for (let i = 0; i < filterKeys.length; i++) {
       const filter = filters[filterKeys[i]];
-      if (filter.operator !== 'is' && filter.value === null) {
+      if (filter.operator !== 'is' && filter.value === '') {
```

Comment:
Think this should be `(filter.operator !== 'is' && (filter.value === '' || filter.value === null) )`

User can still pass null with `'{{null}}'` at the query editory

Reply:
> Think this should be `(filter.operator !== 'is' && (filter.value === '' || filter.value === null) )`
> 
> User can still pass null with `'{{null}}'` at the query editor. Can rename the function to something like`hasEmptyStringOrNullValue`

Yeah Sure, will make changes like this as well as rename the function.